articles/logs/streams/azure-event-grid.md

```diff
@@ -0,0 +1,106 @@
+
+## Send events from Auth0 to Azure Event Grid
+
+To send Auth0 events to Azure, you'll need to:
+
+1. Enable Event Grid resource provider.
+2. Set up an event source (in this case, this is Auth0).
+3. Set up an event handler, the app or service where the event will be sent.
+
+For more information about these concepts, [see Concepts in Azure Event Grid](https://docs.microsoft.com/en-us/azure/event-grid/concepts)
+
+### Enable Event Grid resource provider
+
+If you haven’t previously used Event Grid, you will need to register the Event Grid resource provider. If you've used Event Grid before, skip to the next section.
+
+In your Azure portal:
+
+1. Select Subscriptions
+2. Select the subscription you’re using for Event Grid
+3. On the left menu, under Settings, select Resource providers
+4. Find Microsoft.EventGrid
+5. Select Register
+6. Refresh to make sure the status changes to Registered
+
+### Set up an Auth0 event source
+
+Part of the integration process is to set Auth0 up for use as an event source (this step happens on your Dashboard).
+
+1. Log in to the [Auth0 Dashboard](${manage_url}).
+2. Navigate to **Logs > Streams**.
+3. Click **+ Create Stream**.
+4. Select **Azure Event Grid** and enter a unique name for your new stream.
+5. On the next screen, provide the following settings for your Event Grid stream:
+
+| Setting | Description |
+|---------|-------------|
+| Name | A unique display name to distinguish this integration from other integrations. |
+| Azure Subscription ID | The subscription ID is the unique alphanumeric string that identifies your Azure subscription. |
+| Azure Region | The region where your Azure subscription is hosted. |
+| Resource Group name | Name of the Azure resource group, which is entity to manage all Azure assets within one subscription. |
+
+6. Click **Save**.
+
+Go to the [Azure Portal](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.EventGrid%2FpartnerTopics) to complete the final steps of the integration.
+
+### Set up an event handler
+
+Go to your Azure subscription and spin up a service that is supported as an event handler (for a full list of all supported event handlers go to this [article](https://docs.microsoft.com/en-us/azure/event-grid/event-handlers)).
+
+#### Activate your Auth0 Partner Topic in Azure
+
+Activating the Auth0 topic in Azure allows events to flow from Auth0 to Azure.
+
+1. Log in to the [Azure Portal](https://portal.azure.com/).
+2. Search `Partner Topics` at the top and click `Event Grid Partner Topics` under services.
+3. Click on the topic that matches the stream you created in your Auth0 Dashboard.
+4. Confirm the `Source` field matches your Auth0 account.
+5. Click Activate.
+
+#### Subscribe to your Partner Topic
+
+You subscribe to an event grid topic to tell Event Grid which events to send to which event handler.
```

Comment:
This line sounds like of weird. Is this supposed to be imperative voice? Maybe just:
```suggestion
Subscribe to an Event Grid topic to tell Event Grid which events to send to your event handler.
```